apiserver/plane/api/views/project.py

```
@@ -158,101 +248,16 @@ def post(self, request, slug):
-                            workspace=serializer.instance.workspace,
-                            group=state["group"],
-                            default=state.get("default", False),
-                            created_by=request.user,
-                        )
-                        for state in states
-                    ]
-                )
-
+                self.create_project(
+                    request.META.get("HTTP_ORIGIN"), 
+                    user, 
```

Ruff (0.8.2)

Undefined name `user`

(F821)